sycl/test-e2e/Basic/interop/interop_all_backends.cpp

```diff
@@ -1,4 +1,4 @@
-// XFAIL: any-device-is-opencl, any-device-is-cuda, (windows && any-device-is-level_zero)
+// XFAIL: any-device-is-opencl, any-device-is-cuda, (windows && any-device-is-level_zero), gpu-intel-dg2, hip_amd
 // XFAIL-TRACKER: https://github.com/intel/llvm/issues/15819
```

Comment:
Please update the corresponding GH issue: https://github.com/intel/llvm/issues/15819 with the new failures on DG2 and HIP. It would be helpful to add links to the failing GHA job in the issue as well.

Reply:
Ok, done